litmus-portal/frontend/src/components/WorkflowStepper/index.tsx

```diff
@@ -477,11 +478,15 @@ const CustomStepper = () => {
               </ButtonOutline>
               {activeStep === steps.length - 1 ? (
                 <ButtonFilled
-                  isDisabled={validateWorkflowName(name)}
+                  isDisabled={validateWorkflowName(name) || loading}
```

Comment:
can't we use the buttons from litmus-ui or do you want to tackle all that in a separate PR?

Reply:
I'm updating in the same PR. Thanks for pointing this out. @arkajyotiMukherjee

Reply:
The only thing is... it's used in a lot of places...do you want to do all that in this PR?